### ISSUE TITLE
How to get spherical effect

### ISSUE THREAD

**Issue by abhilash-malkar** [bgstaal/multipleWindow3dScene#7]
How to get spherical effect like this

![IMG_1043](https://github.com/bgstaal/multipleWindow3dScene/assets/91771738/5059a560-681e-4c46-8a2a-d1f1e23a69b8)

**Comment by loneicewolf** [comment]
I agree :)  will watch this convo closely 👍🏼

**Comment by EstebanMAP** [comment]
Wanna also, the code are with 2 squares, wanna to know how can we do this spherical effect.
![imagem_2023-11-24_114236291](https://github.com/bgstaal/multipleWindow3dScene/assets/139181148/77cfdc3e-d194-48ee-8b02-c789808679f4)

**Comment by ShinoKana** [comment]
https://github.com/ShinoKana/multipleWindow3dScene
<img width="976" alt="cba819282e1fc915f926f2ed45a97e3" src="https://github.com/bgstaal/multipleWindow3dScene/assets/100673497/9f727a63-ffd0-4eae-a295-23a70862d40c">

**Comment by P3trol** [comment]
Amazing code!

**Comment by EstebanMAP** [comment]
ye amazing, good job, but if someone still have the code who him post on instagram, let us know

**Comment by loneicewolf** [comment]
> https://github.com/ShinoKana/multipleWindow3dScene <img alt="cba819282e1fc915f926f2ed45a97e3" width="976" src="https://private-user-images.githubusercontent.com/100673497/285539647-9f727a63-ffd0-4eae-a295-23a70862d40c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA4NTgwMzAsIm5iZiI6MTcwMDg1NzczMCwicGF0aCI6Ii8xMDA2NzM0OTcvMjg1NTM5NjQ3LTlmNzI3YTYzLWZmZDAtNGVhZS1hMjk1LTIzYTcwODYyZDQwYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyNFQyMDI4NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNDJiNWZlMzA5ZDk0YzM4Y2VjMjhjOWM4ZDBjOWMzM2FkYzM1YjQzOWMzMGVhN2JmMWI3YjM5YjI5YjNiZDZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zcsEs2_j7gm1fzU7udDFSs-JRoHCgltrsBmnwDnaoxQ">

thanks for posting this

**Comment by Audran-wol** [comment]
How do you get these spheres and the #atoms same colors any source code for that??

**Comment by Runtrons** [comment]
Yes, PLEASE. I need the code it’s so cool! I’ll literally pay a couple bucks.

**Comment by CypherpunkSamurai** [comment]
Reference https://www.instagram.com/reel/Cz827m0tYy1/?igshid=MzRlODBiNWFlZA==

**Comment by abhilash-malkar** [comment]
but there is no effect like shown in instagram

**Comment by jiangjiang2754** [comment]
.ball {
width: 150px;
height: 150px;
border-radius: 50%;
position: relative;
transform-style: preserve-3d;
animation: spin 5s linear infinite;
}
.ball::before, .ball::after {
content: "";
position: absolute;
top: 0;
left: 0;
width: 150px;
height: 150px;
border-radius: 50%;
background-color: #c1c1c1;
box-shadow: 5px 5px 5px rgba(0,0,0,0.4);
}
.ball::before {
transform: rotateY(90deg) translateX(75px);
}
.ball::after {
transform: rotateX(90deg) translateY(75px);
}
@keyframes spin {
0% {
transform: rotateX(0deg) rotateY(0deg);
}
100% {
transform: rotateX(90deg) rotateY(360deg);
}
}
The effect of the above code is a three-dimensional gray sphere, constantly rotating.

**Comment by hojooo** [comment]
@jiangjiang2754
How do apply this css code?

**Comment by Runtrons** [comment]
> @jiangjiang2754 How do apply this css code?

I don’t think that’s the same thing. But, I would ask chat gpt.

**Comment by Nickeldon** [comment]
> > @jiangjiang2754 How do apply this css code?
> 
> I don’t think that’s the same thing. But, I would ask chat gpt.

Don't use ChatGPT. It will just make your life harder (in terms of coding). Rather start to learn web developement

**Comment by loneicewolf** [comment]
> > > @jiangjiang2754 How do apply this css code?
> > 
> > 
> > I don’t think that’s the same thing. But, I would ask chat gpt.
> 
> Don't use ChatGPT. It will just make your life harder (in terms of coding). Rather start to learn web developement

agree here, it's a bit better to use manual coding than a bot, even if the bot would produce, a actual good code, it's better to learn it `:)`

**Comment by Lobohombo** [comment]
En el idioma de mi país por favor
México

Único dueñoDarioRodolfo

**Comment by loneicewolf** [comment]
english? sorry :o

**Comment by Lobohombo** [comment]
Gracias pero soy de tercera edad y no se leer su página web


Único dueñoDarioRodolfo

**Comment by ronangrant** [comment]
> > > @jiangjiang2754 How do apply this css code?
> > 
> > 
> > I don’t think that’s the same thing. But, I would ask chat gpt.
> 
> Don't use ChatGPT. It will just make your life harder (in terms of coding). Rather start to learn web developement

I asked it to recreate the orb (From a single image), this is what GPT4 did first try:
<img width="494" alt="Screenshot 2023-11-26 at 9 06 52 PM" src="https://github.com/bgstaal/multipleWindow3dScene/assets/67560747/f780a740-8447-4e34-a5e8-01c4b538d4b6">

If you truly think that then its because you have not tried GPT4 or you do not know how to use it.

**Comment by loneicewolf** [comment]
> > > > @jiangjiang2754 How do apply this css code?
> > > 
> > > 
> > > I don’t think that’s the same thing. But, I would ask chat gpt.
> > 
> > 
> > Don't use ChatGPT. It will just make your life harder (in terms of coding). Rather start to learn web developement
> 
> I asked it to recreate the orb (From a single image), this is what GPT4 did first try: <img alt="Screenshot 2023-11-26 at 9 06 52 PM" width="494" src="https://private-user-images.githubusercontent.com/67560747/287079947-
> 
> If you truly think that then its because you have not tried GPT4 or you do not know how to use it.

I think he just meant generally speaking, that is - don't recommend to use GPT  because the code or output might be bad in some way that we dont know or is hard to identify (note, I am not saying it's not good to use it, im saying just be careful) :)

**Comment by ruijieren98** [comment]
We also tried to develop the code toward 3D effect for a bit. Check it here https://github.com/ruijieren98/multipleWindow3dScene

**Comment by cirovargas** [comment]
> Reference https://www.instagram.com/reel/Cz827m0tYy1/?igshid=MzRlODBiNWFlZA==

we were deceived 😂😂😂
